src/shared/symbols.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+import Symbol from 'es-symbol'
+
+export const ACTION_HANDLER = Symbol('action creator handler')
```

Comment:
Not to my knowledge.

The src files that gets built are `src/alt.js` or `src/alt-with-addons.js`.  Both of these may import numerous things, but all that is accessible is what they export for use, which is just Alt.

Essentially it is why we need this line in the addons file:
```js
Alt.addons = {
  ActionListeners,
  AltContainer,
  AltManager,
  DispatcherRecorder,
  atomicTransactions,
  chromeDebug,
  makeFinalStore,
  withAltContext
}
```
because importing the addons does not make them available so we have to attach them to the Alt object that we are exporting.